components/checkbox/stories/template.js

```diff
@@ -91,3 +91,83 @@ export const Template = ({
 		</label>
 	`;
 };
+
+/* Shows multiple checkboxes in various states of selected, unselected, indeterminate, etc. */
+export const DocsCheckboxGroup = (args, context) => Container({
+	direction: "column",
+	withBorder: false,
+	content: html`
+		${Template({
+			...args,
+			context,
+			iconName: undefined,
+		})}
+		${Template({
+			...args,
+			context,
+			isChecked: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isIndeterminate: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isDisabled: true,
+		})}
+		${Template({
+			...args,
+			context,
+			label: "Checkbox with a long label that should wrap.",
+			customStyles: { "max-inline-size": "200px" },
+		})}
+	`
+});
+
+/* This template group showcases multiple CheckboxGroups in various states of disabled, read-only, invalid, etc. */
+export const AllVariantsCheckboxGroup = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			withBorder: false,
+			direction: "column",
+			heading: "Default",
+			content: DocsCheckboxGroup(args, context)
+		})}
+		${Container({
+			withBorder: false,
+			direction: "column",
+			heading: "Invalid",
+			content: html`
+				${DocsCheckboxGroup({
+					...args,
+					isInvalid: true,
+				}, context)}
+			`
```

Comment:
```suggestion
			content: DocsCheckboxGroup({ ...args, isInvalid: true }, context)
```

Note the others below that also match this format

Reply:
That's what I get for committing from the GitHub UI 🤦‍♀️ Thank you!

checkbox: https://github.com/adobe/spectrum-css/pull/2793/commits/18c162a4857c872dbfc729a21b0580ebab9ae58e
coach indicator: https://github.com/adobe/spectrum-css/pull/2793/commits/36f02b7eca12830aa074b7be95ed1d303f453c8c